pex/pex_builder.py

```diff
@@ -94,7 +94,8 @@ def __maybe_run_venv__(pex, pex_root, pex_path):
     return
 
   TRACER.log('Executing venv PEX for {{}} at {{}}'.format(pex, venv_pex))
-  __re_exec__(venv_pex)
+  venv_python = os.path.join(venv_home, 'bin', 'python')
```

Comment:
Facepalm. I'm pretty sure I never really needed to invent the short venv symlinks - this pretty much covers things.

Reply:
Well, I guess it doesn't. The venv `bin/` console scripts are still used by folks building PEXes with `--venv {prepend,append}`. Those folks leverage the fact the console scripts are put on the PATH by Pex to call them by name (and not full path), and that utilizes the short venv symlink shebang we write into those console scripts. So that still has value.